src/index.js

```diff
@@ -248,4 +252,37 @@ IdTokenVerifier.prototype.decode = function (token) {
   };
 };
 
+/**
+ * @callback validateAccessTokenCallback
+ * @param {Error} [err] error returned if the validation cannot be performed
+ * or the token is invalid. If there is no error, then the access_token is valid.
+ */
+
+/**
+ * Validates an access token based on {@link http://openid.net/specs/openid-connect-core-1_0.html#ImplicitTokenValidation}.
+ * The id_token should be decoded and verified before using this function
+ *
+ * @method validateAccessToken
+ * @param {String} access_token the access_token
+ * @param {String} alg the algorithm used to hash the id_token.
+ * @param {String} atHash the `at_hash` value of the id_token
+ * @param {validateAccessTokenCallback} cb callback used to notify the results of the validation.
+ */
+IdTokenVerifier.prototype.validateAccessToken = function (accessToken, alg, atHash, cb) {
+  if (supportedAlgs.indexOf(alg) === -1) {
```

Comment:
I think you should be checking against `options.expectedAlg`, since that's the algorithm defined at built stage and it's already being checked against `supportedAlgs` right there. https://github.com/auth0/idtoken-verifier/blob/a998eadfcdf9265ae7518dccd66c3e6164b14f8d/src/index.js#L22-L23

Reply:
fixed